src/context/reducers/notificationReducer.tsx

```diff
@@ -10,40 +10,44 @@ export enum NotificationItemVariant {
 }
 
 export interface NotificationItem {
-  variant: 'success' | 'danger' | 'warning' | 'info' | 'default'
-  title: string;
-  message: string;
-  timestamp: Moment;
+  variant: 'success' | 'danger' | 'warning' | 'info' | 'default', //here need? how does it work? what's it doing? remove?
```

Comment:
Please, remove this comment if it has been resolved. If not and further investigation/work is required, convert this PR into draft.

Reply:
apologies, corrected this in a later PR, but removed in this one as well